conf/spark-defaults.conf

```diff
@@ -0,0 +1,18 @@
+spark.armada.jobSetId armada-spark
+spark.executor.instances 3
+spark.jars.ivy /tmp/.ivy
+spark.armada.scheduling.nodeUniformity armada-spark
```

Comment:
I think for defaults it should work on any k8s cluster or armada cluster.
This requires config in both k8s deployment and Armada.

I'd remove node uniformity and node selectors for defaults